Source/WTF/wtf/PlatformUse.h

```diff
@@ -297,10 +297,10 @@
 #endif
 
 #if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110000) \
```

Comment:
Same, I think this also collapses to COCOA.

Reply:
The lines below do `&& PLATFORM(IOS_FAMILY_SIMULATOR)` so I don't think that's the same as `COCOA`.